.tekton/build-pipeline.yaml

```diff
@@ -0,0 +1,594 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
```

Comment:
Shouldn't it be PipelineRun?

Reply:
No, the pipeline is referenced by the push and pull pipelineruns

Reply:
No, MintMaker still updates these just fine. For example, https://github.com/konflux-ci/oras-container/pull/234/files